src/xenia/kernel/xam/xam_nui.cc

```diff
@@ -88,6 +385,86 @@ dword_result_t XamShowNuiHardwareRequiredUI_entry(unknown_t unk1) {
 }
 DECLARE_XAM_EXPORT1(XamShowNuiHardwareRequiredUI, kNone, kImplemented);
 
+dword_result_t XamShowNuiGuideUI_entry(unknown_t unk1, unknown_t unk2) {
+  /* Notes:
+   - calls an unnamed function that checks XamNuiHudIsEnabled and
+   XamNuiHudSetEngagedTrackingID
+     - if XamNuiHudIsEnabled returns false then fuctions fails return
+   X_ERROR_ACCESS_DENIED
+     - else calls XamNuiHudSetEngagedTrackingID and if returns less than 0 then
+   returns X_ERROR_FUNCTION_FAILED
+     - else return X_ERROR_SUCCESS
+   - if return offunc is X_ERROR_SUCCESS then call up ui screen
+   - else return value of func
+  */
+
+  // decompiler error stops me from knowing which param gets used here
+  uint32_t result = XeXamNuiHudCheck(0);
+  if ((result & 0xffffffff) == 0) {
+    // operations here
+    // XMsgSystemProcessCall(0xfe,0x21030, undefined local_30[8] ,0xc);
+  }
+  return result;
+}
+DECLARE_XAM_EXPORT1(XamShowNuiGuideUI, kNone, kStub);
+
+/* XamNuiIdentity Notes:
+   - most require message calls to xam in 0x0002Cxxx area
+*/
+uint64_t NUI_Session_Id = 0;
+
+qword_result_t XamNuiIdentityGetSessionId_entry() {
+  if (NUI_Session_Id == 0) {
+    std::memset(&NUI_Session_Id, 0xFD,
```

Comment:
Instead of using memset to fill it just use ``NUI_Session_Id = 0xFDFDFDFDFDFDFDFD;`` or even better ``0xDEADF00DDEADF00D``